foreshadow/tests/test_configs/invalid_transformer_params.json

```diff
@@ -1,27 +1,32 @@
 {
   "columns":{
-    "crim":["TestGenericIntent",
-            [
-              ["StandardScaler", "Scaler", {"with_mean":false}]
-            ]],
-    "indus":["TestGenericIntent"]
+    "crim":{
+	    "intent": "TestGenericIntent",
+            "pipeline":[
+	    {"transformer": "StandardScaler", 
+	       "name": "Scaler", 
+	       "parameters": {"with_mean":false}
+	    }]},
+    "indus":{"intent": "TestGenericIntent"}
   },
 
   "postprocess":[
-    ["pca",["crim","indus","nox"],[
-      ["PCA", "PCA", {"n_components":4}]
-    ]]
+	  {"BAD": "pca",
```

Comment:
Will this "BAD" cause another error other than the one intended by this configuration?

Reply:
My bad. It apparently catches the correct error first but you are correct that would throw a KeyError down the line if not caught beforehand.